src/microbe_stage/MicrobeHUD.cs

```diff
@@ -242,7 +242,9 @@ public override void _Ready()
         populationLabel = GetNode<Label>(PopulationLabelPath);
         patchLabel = GetNode<Label>(PatchLabelPath);
         editorButton = GetNode<TextureButton>(EditorButtonPath);
-        helpScreen = GetNode<Control>(HelpScreenPath);
+        helpScreen = GetNode<HelpScreen>(HelpScreenPath);
+
+        helpScreen.BuildHelpTexts("MicrobeStage");
```

Comment:
I think that making this an editor configurable property would be nice. And then load the help texts in the help screen _Ready method.